prime-dotnet-webapi/Models/EnrolleeLinkedEnrolments.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
```

Comment:
Be careful of this auto-import. The app uses `Newtonsoft.Json`, not `System.Text.Json.Serialization`

Reply:
I'm not immune to this mistake, btw 😅.

Reply:
hahaha updated